### ISSUE TITLE
Reachable variables in the commands

### ISSUE THREAD

**Issue by ruben-arts** [prefix-dev/pixi#28]
I was thinking of a custom command to update versions of all files in the project but for that i would need the version of the `pax.toml`

What would be nice if you could access the project files and possibly some more things in your commands. 
eg.
```toml
[project]
name = "pax"
version = "0.1.0"
description = "Add a short description here"
authors = ["Ruben Arts <ruben@prefix.dev>"]
channels = ["conda-forge"]
platforms = ["linux-64", "win-64", "osx-64", "osx-arm64"]

[commands]
custom_command = "echo project_name: $PROJECT_NAME version: $PROJECT_VERSION desc: $PROJECT_DESCRIPTION"
```
and possibly more info about the config you give.

**Comment by ruben-arts** [comment]
#78 Would solve this but might not be the best solution for the toml file itself as we could do something smarter with that.

**Comment by baszalmstra** [comment]
Yeah I think it would be nice to introduce minijinja syntax in commands etc. That way you could write

```toml
[commands]
config = "cmake -S ${{ project.root }} -B ${{ project.root }}/build"
```

**Comment by baszalmstra** [comment]
We have implemented the deno shell (#173 ) and meta variables (#78 ) which should cover most of the things discussed here. Lets open up another issue if we need specific features.
